### PR TITLE
Yusuke day 1 tlc

### PR DESCRIPTION
**Pull request by yk1512**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 1]
```diff
@@ -1,22 +1,22 @@
 <Page1>
 ...No, I'll wait here.
-It's reckless to walk around right after coming back.
+Walking around right after coming back would make me seem overly on edge. // 帰ってきたばかりで歩き回るのはさすがに落ち着きがない
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:3]: where is "seem" coming from? Maybe:
> Walking around right after coming back wouldn't help me calm down

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 I should greet Akiha before I do anything else.
 She's the current head of the Tohno family, after all.
 <Page2>
 I spend some time sitting on the sofa.
 The view of the yard from the window matches what I remember.
 There is a courtyard the size of four tennis courts, and a garden at the far end that seems ridiculous to even measure.
-On the other side of the garden, there are more woods, and I think――――
+On the other side of the garden, there are more woods, and if I'm not mistaken―――― // 庭園の向こうにはさらに林があって、たしか――― would like to reword 'more woods' but not sure how.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:10]: I think "more woods" is fine

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 3]
```diff
@@ -28,33 +28,33 @@ We've seen each other a few times when you were a kid, albeit not like this. Eve
 <Page7>
 Saiki...?
 I wonder if there is any branch of the Tohno family with such a name.
-No, before that―――this person is, how should I say―――
+No, before that―――this person is, how do I say it―――
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:31]: I localized this as:
> No, what I want to know before that―――this person, how should I say it―――//　いや、それ以前に―――この人物は、なんというか―――

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 4]
```diff
@@ -28,33 +28,33 @@ We've seen each other a few times when you were a kid, albeit not like this. Eve
...
 <Page8>
 "It's alright, I'm used to being looked at strangely.
 I burned my face in an accident. For a long time now, I haven't been able to go outside without wearing them. Sunlight is especially harsh.
-Akiha also dislikes me, but that can't be helped. She is also partly responsible for this."
+Akiha also considers me to be a nuisance but that can't be helped. She is also partly responsible for this after all." // 秋葉にも<煙|けむ>たがられているが、こればかりは仕方のない事だ。彼女にも責任の一端はある事だしね
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:35]: I translated this as
> Akiha also dislikes me, but c'est la vie. She is also partly responsible for this, you know."

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 5]
```diff
@@ -28,33 +28,33 @@ We've seen each other a few times when you were a kid, albeit not like this. Eve
...
+About to impulsively stand up, I checked myself, leaning back deeply into the sofa.
 There is obvious contempt and provocation in Saiki's eyes, but there is no point in making a scene here. After all, there is no mistake in what he said.
 <Page13>
 "Oh. Don't you have any objections?"
 "No, it's true that I abandoned my duty.
 But don't you think it's shameful for you to use words like 'deserted' and 'investment' when talking about family?"
 <Page14>
-"I chose those words exactly because we were talking about family.
-Fostering is an investment in the family's future. Apart from the city dwellers, individual freedom does not exist in a chosen bloodline or in a noble family line.
+"I chose those words precisely because we were talking about family.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:56]: I honestly prefer "exactly" here but I guess its a matter of style. we should decide on one and introduce it in both files.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 6]
```diff
@@ -28,33 +28,33 @@ We've seen each other a few times when you were a kid, albeit not like this. Eve
...
+Fostering is an investment in the family's future. Setting aside the commoners, individual freedom does not exist in a chosen bloodline or in a noble family line.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:57]: Here's my version
> Raising a child an investment in the family's future. Setting aside the commoners, humans don't have the freedom to pick their bloodline or nobility.//　養育は一族の将来への投資だ。<市井|しせい>の人間はともかく、選ばれた血筋、尊い家系において、個人の自由は存在しない。Although Shiki is adopted, 'fostering' isn't what he means here.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 7]
```diff
@@ -67,20 +67,20 @@ An outsider...?
 Then this man is not from a branch of the family?
 <Page18>
 "The Saiki family has been on an equal footing with the Tohno family since my grandfather's generation. In modern terms, we'd call them business partners.
-I'm only here on business today. It may have dragged on for a while, but I got a satisfying response. I'm heading home in a good mood.
-Aah, the last thing I want to do is spoil your homecoming with my creepy appearance. Please, excuse me."
+I'm only here on business today. It may have dragged on for a while, but I got a satisfying response. I'll be heading home in a good mood.
+Ah, spoiling your homecoming with my creepy appearance is the last thing I'd want to do. Please, excuse me."
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:71]: I prefer my version 
> Aah, the last thing I want to do is spoil your homecoming with my gloomy appearance. Please, excuse me."//　ああ、この陰気な姿でせっかくの帰郷に水を差すのも忍びない。今日はこのあたりで失礼するよ」

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 8]
```diff
@@ -67,20 +67,20 @@ An outsider...?
...
 The man with the black bandages... leaves the living room with an ironic smile on his face.
 <Page20>
 "......."
-I bit my lip when Saiki said the word 'deserter' to me.
+I bit my lip as I recall Saiki calling me a 'deserter'.
 It is a very accurate description of my position in this mansion, just as he had said.
 <Page21>
 "Excuse me."
-Exactly ten minutes later, the girl comes back.
+Exactly ten minutes later, the girl has returned.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:80]: I prefer my version here.
> After exactly ten minutes have elapsed, the girl returns.//　本当にきっかり10分後、彼女は戻ってきた。 "after 10 minutes" makes it sounds like 10 minutes after Saiki left.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 9]
```diff
@@ -67,20 +67,20 @@ An outsider...?
...
 <Page22>
 "...? Shiki-sama, is there something wrong?"
-It seems she noticed that I've been feeling down after hearing Saiki Gouto's words.
+It seems she's noticed that I've been feeling down after hearing Saiki Gouto's words.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:83]: I think revert

**Review by Avuxo — APPROVED** [review]
I forgot this had so much overlap... we shouldn't have done this separately but too late.
I took all of your overlapping changes I liked and ported them back into my PR. I commented on all the ones I prefer from mine. I suggest you do the same before merging this

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 10]
```diff
@@ -1,22 +1,22 @@
...
 I spend some time sitting on the sofa.
 The view of the yard from the window matches what I remember.
 There is a courtyard the size of four tennis courts, and a garden at the far end that seems ridiculous to even measure.
-On the other side of the garden, there are more woods, and I think――――
+On the other side of the garden, there are more woods, and if I'm not mistaken―――― // 庭園の向こうにはさらに林があって、たしか――― would like to reword 'more woods' but not sure how. 
 <Page3>
 "Oh. I thought I'd stop by before leaving, but I didn't know we had a prior guest."
 The sudden voice made me turn around and gasp.
-In front of the door to the hallway that connects to the lobby,
+In front of the door that connects the hallway to the lobby,
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:14]: I don't think this is correct - 
> ロビーに通じる 廊下 側の 扉の 前には、

Reads to me as "The door on the side of the room adjoining the (corridor that connects to the lobby)", which is a heinous sentence but means something a bit different. This current version suggests that the door Saiki came thorough is between the hallway and lobby, but we are currently in the living room

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt [hunk 11]
```diff
@@ -28,33 +28,33 @@ We've seen each other a few times when you were a kid, albeit not like this. Eve
...
 "I'm talking about you, Shiki-kun. You broke Makihisa's orders and escaped from the mansion, and as a result, you were involved in a stupid accident and deserted."
 <Page11>
-"Deserted. Yes, deserted. You who threw away your parent's investment and abandoned your duty, yet here you are, back at the mansion, trying to act like Akiha's brother.
-Don't you feel ashamed of yourself?"
+"Deserted. Yes, deserted. You discarded your parent's investment and abandoned your duty. Yet here you are, back at the mansion, trying to act like Akiha's brother.
+Do you not feel ashamed of yourself?"
 <Page12>
 "――――――"
-I was just about to impulsively stand up but checked myself, before leaning my back deeply against the sofa.
+About to impulsively stand up, I checked myself, leaning back deeply into the sofa.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:49]: Check myself present tense?

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt (hunk 7 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:71]: weirdly enough, when I first read it last night, it sounded like saiki was saying he wanted to spoil his homecoming before leaving.

File: script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt (hunk 10 above)
Comment (yk1512) [reply script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt:14]: will revert lol

**Pull request merged**